### ISSUE TITLE
More than 1000 characters stroke are composed full of 0 in chinese_stroke.txt.

### ISSUE THREAD

**Issue by MEIZU16** [Hxyz-123/Font-diff#20]
**like this：**
鹤 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 
憨 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 
缭 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 

Is this a special way of handling it, or just missing data?

**Comment by Hxyz-123** [comment]
Yes. These are the missing data. But they can be treated as characters without using stroke conditions in our method.

**Comment by zhenzi0322** [comment]
怎么表示的呢，假设是国字，如果表示呢，我想搞明白这种是怎么添加的。

**Comment by MEIZU16** [comment]
论文里有介绍规则

**Comment by zhenzi0322** [comment]
好的，请教一下，针对英文和标点符号生成的效果如何？
